### ISSUE TITLE
Updated fork - with some PRs 

### ISSUE THREAD

**Issue by eyJhb** [mtoyoda/sl#60]
I have forked this repo and merged some of the PRs I found the most interesting and relevant.
If you feel like anything else should be included or changed, please do make an issue with the fork and or do a new PR! 

[here](https://github.com/eyJhb/sl)

Thanks!

**Comment by ddugovic** [comment]
Thanks!  (I know there's no CLA, but I happily consent to this.)

**Comment by eyJhb** [comment]
@ddugovic yeah, didn't really think of that! But I am hoping that the PRs I merged are okay ;) 
But no clue if people will adapt to it! I need to make some changes to it too..

**Comment by PartialVolume** [comment]
When the maintainer prefers to leave the code as is, there's certainly no harm in forking, in fact I'd recommend it.

As for CLAs [https://ben.balter.com/2018/01/02/why-you-probably-shouldnt-add-a-cla-to-your-open-source-project/](https://ben.balter.com/2018/01/02/why-you-probably-shouldnt-add-a-cla-to-your-open-source-project/) 

That article suggests projects are better if without them.

**Comment by buhtz** [comment]
It seems there is no reaction anymore from Toyoda-san.
I would suggest you to treat your repo `eyJhb/sl` as the "official one" and go on with developing and fixing. The GNU Linux distros will follow then.

**Comment by eyJhb** [comment]
Given the amount of issues/PRs that are open for sl (on my fork), I'm not sure it's a good idea that I maintain the fork anymore. Most likely someone with more time, that still uses sl would be better.

**Comment by buhtz** [comment]
In the end it doesn't matter if it is your fork, this repo (you take over) or another fork. Important is that something is rolling. 😄 
But OK, thanks for your feedback.

IMHO in the first place it is important that the maintainer of this repo state in the README the current situation. The repo is dead (and this is OK).

**Comment by gresm** [comment]
Seems like someone picked up your fork: https://github.com/achernetsov/sl

**Comment by gresm** [comment]
Seems like someone [picked up](https://github.com/vrtx-mortem/sl) your fork.

**Comment by eyJhb** [comment]
Oh nice! Hopefully they are better at it than me :D

**Comment by eyJhb** [comment]
> Seems like someone [picked up](https://github.com/vrtx-mortem/sl) your fork.

It's a shame that the repo does not allow issues, unsure why however.
